torch/_torch_docs.py

```diff
@@ -7754,3 +7754,82 @@ def merge_dicts(*dicts):
     >>> g_cpu.device
     device(type='cpu')
 """)
+
+add_docstr(torch.searchsorted,
+           r"""
+searchsorted(sorted_sequence, values, out_int32=False, right=False, out=None) -> Tensor
+
+Returns a new tensor, same size as :attr:`values`, containing the result of applying
+either lower_bound or upper_bound (depends on :attr:`right`) to each value in :attr:`values`
+against the corresponding values on the innermost dimension of :attr:`sorted_sequence`.
+
+Args:
+    sorted_sequence (Tensor): N-D or 1-D tensor, containing sorted sequence on innermost dimension.
+      If :attr:`sorted_sequence` is 1-D tensor, then every value in :attr:`values` will apply 
+      lower_bound or upper_bound against the entire :attr:`sorted_sequence`.
+    values (Tensor or Scalar): N-D tensor or a Scalar containing the search value(s).    
+    out_int32 (bool): output data type is torch.int32 if True, torch.int64 otherwise.
+    right (bool): apply upper bound if True, lower bound otherwise.
+    out (Tensor, optional): the output tensor, must be the same size as :attr:`values` if provided.
+
+.. note:: If your use case is always 1-D sorted sequence, :func:`torch.bucketize` is preferred,
+          it has less internal check, slightly better performance. 
```

Comment:
nit: "If your use case is always 1-D sorted sequence, :func:`torch.bucketize` is preferred because it has slightly better performance" -> we don't really have to tell users about the internal details

Reply:
I think we should give user some level of the internal details on why we recommend one api over another, this will make people more comfortable to switch. Numpy doc did this as well.
https://docs.scipy.org/doc/numpy/reference/generated/numpy.digitize.html ->
"Note that as the order of the arguments are reversed, the side must be too. The searchsorted call is marginally faster, as it does not do any monotonicity checks."